### ISSUE TITLE
function args comma style

### ISSUE THREAD

**Issue by MrOutput** [eslint/eslint#6961]
linter does not enforce comma style for function arguments

**What version of ESLint are you using?**

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**

**What did you do? Please include the actual source code causing the issue.**

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**

**Comment by platinumazure** [comment]
We're going to need those questions answered. :smile:

**Comment by eslintbot** [comment]
Hi @MrOutput, thanks for the issue. It looks like there's not enough information for us to know how to help you.

If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

Requesting a rule change? Please see [Proposing a Rule Change](http://eslint.org/docs/developer-guide/contributing/rule-changes) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!

**Comment by MrOutput** [comment]
a function's arg list should be forces to the same style as objects (comma-first or comma-last). for example

```js
array.splice(
  0
, 1 //SHOULD BE ENFORCED COMMA FIRST
)
```

**Comment by vitorbal** [comment]
Hi @MrOutput, thanks for getting back to us. We kindly ask everyone that would like to request an enhancement to a rule or a new rule to fill in the template that the @eslintbot [mentions above](https://github.com/eslint/eslint/issues/6961#issuecomment-241996698).

Could you please take a moment to do that? That will help the team understand what you are proposing and allow us to give quicker feedback on your proposal. Thanks a lot!

**Comment by vitorbal** [comment]
@MrOutput one last ping, could you take the time to answer the questions from the @eslintbot above? Otherwise I'm afraid we can't help much and I'm just going to close this. Thanks!

**Comment by MrOutput** [comment]
no thanks
